src/types/markers.js

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I was having trouble finding marker information, so I split this out into another type file.